pkg/auth/peer.go

```diff
@@ -203,12 +206,14 @@ func New(ctx context.Context, log *zap.Logger, config Config, configDir string)
 // LogRequests logs requests.
 func LogRequests(log *zap.Logger, h http.Handler) http.Handler {
 	return whroute.HandlerFunc(h, func(w http.ResponseWriter, r *http.Request) {
+
```

Comment:
nit: try to reduce these whitespace changes for code that isn't being changed, or make them a separate commit